E.Deezer/Api/Album.cs

```diff
@@ -19,7 +19,7 @@ public interface IAlbum : IObjectWithImage
         string Link { get; }
         string Title { get; }
         string Label { get; }
-        uint GenreId { get; }
+        int GenreId { get; }
```

Comment:
Think we should change all `GenreId` properties to be `int` to keep everything consistant

Reply:
I ran a quick search on the Entire Solution looking for `uint GenreId` but found nothing.

`Find all "uint GenreId", Subfolders, Find Results 1, Entire Solution, ""
  Matching lines: 0    Matching files: 0    Total files searched: 117`